### PR TITLE
patrolling and a few function of enemie weapon

### PR DESCRIPTION
**Pull request by DimaDiD**

### CODE REVIEW

File: Assets/Enemies/GunEnemy.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+            Target target = hit.transform.GetComponent<Target>();
+            if (target != null)
+            {
+                target.enemiesTakeDamage(damage);
+            }
+
+            Instantiate(muzzleFlash, attackPos.position, Quaternion.identity);
+            Invoke("ShotReset", timeBetweenShooting);
+        }*/
```
Comment (OlegXXL) [inline Assets/Enemies/GunEnemy.cs:49]: Why this comment?

**Review by loburyaka — COMMENTED** [review]
Need description.

File: ProjectSettings/TagManager.asset [hunk 2]
```diff
@@ -13,8 +23,8 @@ TagManager:
   - UI
   - Ground
   - 
-  - 
-  - 
+  - Enemie
```
Comment (loburyaka) [inline ProjectSettings/TagManager.asset:26]: Learn English.

**Review by SwaTiK4931 — APPROVED** [review]

File: Assets/Enemies/GunEnemy.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+using UnityEngine;
+
+public class GunEnemy : MonoBehaviour
+{
+    //Gun stats
+    public int damage;
+    public float timeBetweenShooting, spread, range, reloadTime;
+ 
+
+    //some bools
```
Comment (OlegXXL) [inline Assets/Enemies/GunEnemy.cs:12]: stop press Enter

**Review by OlegXXL — APPROVED** [review]

**Review by loburyaka — APPROVED** [review]
Everything is OK, but read comments and solve the conflicts.

**Pull request closed without merging**